core/src/main/java/com/graphhopper/routing/DijkstraBidirectionRef.java

```diff
@@ -157,8 +157,8 @@ public boolean finished() {
         return currFrom.weight + currTo.weight >= bestPath.getWeight();
     }
 
-    void fillEdges(SPTEntry currEdge, PriorityQueue<SPTEntry> prioQueue,
-                   IntObjectMap<SPTEntry> bestWeightMap, EdgeExplorer explorer, boolean reverse) {
+    protected void fillEdges(SPTEntry currEdge, PriorityQueue<SPTEntry> prioQueue,
```

Comment:
This should not be necessary IMO

Reply:
If we want to use package private and still use this method (and `pqOpenSetFrom/To`) we need to move `DijkstraBidirectionCHWithSOD` to the same package as `DijkstraBidirectionRef`. In this case I would also move `DijkstraBidirectionCH` and `AStarBidirectionCH` from `PrepareContractionHierarchies` to `com.graphhopper.routing`. I would find this cleaner anyway ;)

Reply:
Yes, please ;)